src/FSharpVSPowerTools.Core/UntypedAstUtils.fs

```diff
@@ -653,3 +653,112 @@ let getModuleOrNamespacePath (pos: pos) (ast: ParsedInput) =
+                        if directoryExists directory then
+                            let includeDirective = ResolvedDirectory(directory)
+                            pushInclude includeDirective
+                            yield Include (includeDirective, range)
+
+                    | SynModuleDecl.HashDirective(ParsedHashDirective("load",files,range),_) ->
+                        for f in files do
+                            if isPathRooted f && fileExists f then
+
+                                // this is absolute reference to an existing script, easiest case
+                                yield Load (ExistingFile f, range)
+
+                            else
+
+                                // I'm not sure if the order is correct, first checking relative to file containing the #load directive
+                                // then checking for undocumented resolution using previously parsed #I directives
+                                let fileRelativeToCurrentFile = baseDirectory </> f
+                                if fileExists fileRelativeToCurrentFile then
+
+                                    // this is existing file relative to current file
+                                    yield Load (ExistingFile fileRelativeToCurrentFile, range)
+
+                                else
+
+                                    // match file against first include which seemingly have it found
+                                    let maybeFile =
+                                        includesSoFar
+                                        |> Seq.tryPick (function
+                                            | (ResolvedDirectory d) ->
+                                                let filePath = d </> f
+                                                if fileExists filePath then Some filePath else None
+                                        )
+                                    match maybeFile with
+                                    | None -> () // can't load this file even using any of the #I directives...
+                                    | Some f ->
+                                        yield Load (ExistingFile f, range)
+
+                    | _ -> ()
+            |]
+
+        match ast with
+        | ParsedInput.ImplFile(ParsedImplFileInput(fn,_,_,_,_,modules,_)) -> parseDirectives modules fn
+        | _ -> [||]
+
+    /// returns the Some (complete file name of a resolved #load directive at position) or None
+    let getHashLoadDirectiveResolvedPathAtPosition (pos: pos) (ast: ParsedInput) : string option =
+        let l = pos.Line
+        let directive =
+            getIncludeAndLoadDirectives ast
+            |> Array.tryFind (function
+                | Load(ExistingFile _, range)
+                  // check the line is within the range
+                  // (doesn't work when there are multiple files given to a single #load directive)
+                  when range.StartLine <= l && range.EndLine >= l
```

Comment:
I think we should use posInRange here (see https://github.com/fsprojects/VisualFSharpPowerTools/blob/c474ac16df14d04b820f8296d83a47439867ca4a/src/FSharpVSPowerTools.Core/InterfaceStubGenerator.fs#L575)